app/src/main/java/com/deveddy/clujbike/data/repository/mapper/DataMapper.java

```diff
@@ -0,0 +1,5 @@
+package com.deveddy.clujbike.data.repository.mapper;
+
+public interface DataMapper<T, V> {
+    V mapFrom(T t);
```

Comment:
T would stand for "to" and F would stand for "from", as I understand. It would simplify the writing. I will change it